config.js

```diff
@@ -7,35 +7,35 @@ function getConfig(env) {
     case 'mainnet':
         return {
             networkId: 'mainnet',
-            nodeUrl: 'https://rpc.mainnet.nearprotocol.com',
+            nodeUrl: 'https://rpc.mainnet.near.org',
             contractName: CONTRACT_NAME,
-            walletUrl: 'https://wallet.mainnet.nearprotocol.com',
-            helperUrl: 'https://helper.mainnet.nearprotocol.com',
+            walletUrl: 'https://wallet.mainnet.near.org',
+            helperUrl: 'https://helper.mainnet.near.org',
         };
     case 'development':
     case 'testnet':
         return {
             networkId: 'default',
-            nodeUrl: 'https://rpc.testnet.nearprotocol.com',
+            nodeUrl: 'https://rpc.testnet.near.org',
             contractName: CONTRACT_NAME,
-            walletUrl: 'https://wallet.testnet.nearprotocol.com',
-            helperUrl: 'https://helper.testnet.nearprotocol.com',
+            walletUrl: 'https://wallet.testnet.near.org',
+            helperUrl: 'https://helper.testnet.near.org',
```

Comment:
`https://helper.testnet.near.org/` not working

Reply:
endpoint now working, but it throws a cert error @frol

Reply:
@chadoh  certs are now handled well.